.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -41,9 +41,9 @@ body:
       label: Product Area
       description: Where in the product did you find the bug?
       options:
-        # Keep in sync with feature.yml and labels.yml
+        # begin product areas - autogenerated by bin/react-to-blah-blah-blah.py
```

Comment:
does `bin/react-to-blah-blah-blah.py` exist though? 👀

Reply:
No it was a joke, sorry. Fixed in 1b5d209eb28b8a8fdcda7997d684dfa417339db0. 👍